### PR TITLE
wasm: fix compile error for wasm

### PR DESCRIPTION
**Pull request by 5kbpers**
### What problem does this PR solve? 
Fix compile error for wasm

### What is changed and how it works?
* move `mathutil` dependency to `util`, so that we do not need to depend on it when compile wasm
*  add signal_wasm

### Check List 

Tests 

 - No code

Code changes

 - Has exported function/method change

Side effects

 - Increased code complexity
 - Breaking backward compatibility

### CODE REVIEW

File: cmd/importer/data.go [hunk 1]
```diff
@@ -19,7 +19,7 @@ import (
 	"sync"
 	"time"
 
-	"github.com/cznic/mathutil"
```
Comment (lonng) [inline cmd/importer/data.go]: Can we remove the dependence `github.com/cznic/mathutil` from `go.mod` to avoid using it in our code base again?

Comment (5kbpers) [reply cmd/importer/data.go]: Seems not, we still need to depend on it in `util/mathutil`.😓

**Comment by 5kbpers** [comment]
PTAL, thx

**Comment by 5kbpers** [comment]
test failed: https://internal.pingcap.net/idc-jenkins/blue/organizations/jenkins/tidb_ghpr_integration_copr_test/detail/tidb_ghpr_integration_copr_test/1220/pipeline

I'm not sure if it's related to my changes. 😓

**Comment by lonng** [comment]
> test failed: https://internal.pingcap.net/idc-jenkins/blue/organizations/jenkins/tidb_ghpr_integration_copr_test/detail/tidb_ghpr_integration_copr_test/1220/pipeline
> 
> I'm not sure whether it's related to my changes. 😓

The `copr-test` is unstable in recently, we are fixing it.

**Pull request merged**